### ISSUE TITLE
Adding support for Elixir

### ISSUE THREAD

**Issue by f-lombardo** [erlide/erlide#251]
It would be really nice to have support for http://elixir-lang.org/

**Comment by vladdu** [comment]
Yes, but most of the IDE support is language specific. Only the libraries for handling the erlang runtimes are common. So it's unfortunately not an easy task; I'd say it requires a separate project.
